src/htmlparser/fsm_config.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
```

Comment:
Why was this removed?

Reply:
See the message of the commit that removes it.

In short: it is not a Python script to run (only a module to import), so it does not need it.